user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Define_New_API.md

```diff
@@ -151,25 +151,30 @@ You can also use the `StatusCode` enum, which contains suggestions, and cast tha
 > - A simple example is included in the [Automation script example](#creating-the-api-automation-script).
 > - For more insight into which HTTP status codes to use in which circumstances, see [HTTP status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-## Creating an API definition and token(s)
+## Creating an API and token(s) in Automation
```

Comment:
Maybe use 'the Automation app' instead of Automation?

Reply:
I would also recommend adjusting this title and the corresponding cross-references. We don't usually refer to "Automation" as such, but to "DataMiner Automation" or "the Automation module".

```suggestion
## Creating an API and token(s) in DataMiner Automation
```